lametro/views.py

```diff
@@ -64,16 +61,13 @@
     LAMetroSubject,
     EventBroadcast,
     BoardMemberDetails,
-    EventNotice,
 )
 from lametro.forms import (
-    AgendaUrlForm,
-    AgendaPdfForm,
     LAMetroCouncilmaticSearchForm,
 )
+from lametro.services import EventService
```

Comment:
I dig this idea of services. Is it mostly because this part of the view was getting chunky?

Reply:
Yep. It keeps the views lean and makes it easier to test the different pieces of information we expect.